From 98be48923b1edd04ca633e14b5aaffd460226918 Mon Sep 17 00:00:00 2001
From: Shengjiu Wang <shengjiu.wang@freescale.com>
Date: Mon, 5 Jun 2017 16:01:04 +0800
Subject: [PATCH 1861/5242] MLK-15004-6: Document: sound: update document for
 audio

commit  1b182fc2448a44bb1e3538b3e142f5ffd6585a4f from
https://source.codeaurora.org/external/imx/linux-imx.git

update compatible string for imx-audio-cs42888.txt and fsl,esai.txt

Signed-off-by: Shengjiu Wang <shengjiu.wang@freescale.com>
Signed-off-by: Meng Li <Meng.Li@windriver.com>
---
 .../devicetree/bindings/sound/fsl,esai.txt         |    1 +
 .../bindings/sound/imx-audio-cs42888.txt           |    4 +++-
 2 files changed, 4 insertions(+), 1 deletion(-)

diff --git a/Documentation/devicetree/bindings/sound/fsl,esai.txt b/Documentation/devicetree/bindings/sound/fsl,esai.txt
index 7387375..0a377ec 100644
--- a/Documentation/devicetree/bindings/sound/fsl,esai.txt
+++ b/Documentation/devicetree/bindings/sound/fsl,esai.txt
@@ -8,6 +8,7 @@ other DSPs. It has up to six transmitters and four receivers.
 Required properties:
 
   - compatible		: Compatible list, must contain "fsl,imx6ull-esai",
+			  "fsl,imx8qxp-v1-esai",
 			  "fsl,imx35-esai" or "fsl,vf610-esai"
 
   - reg			: Offset and length of the register set for the device.
diff --git a/Documentation/devicetree/bindings/sound/imx-audio-cs42888.txt b/Documentation/devicetree/bindings/sound/imx-audio-cs42888.txt
index af746c4..7e5148f 100644
--- a/Documentation/devicetree/bindings/sound/imx-audio-cs42888.txt
+++ b/Documentation/devicetree/bindings/sound/imx-audio-cs42888.txt
@@ -1,7 +1,9 @@
 Freescale i.MX audio complex with CS42888 codec
 
 Required properties:
-- compatible : "fsl,imx-audio-cs42888"
+- compatible : "fsl,imx-audio-cs42888", or
+		"fsl,imx8qm-sabreauto-cs42888" or
+		"fsl,imx8qxp-sabreauto-cs42888"
 - model : The user-visible name of this sound complex
 - esai-controller : The phandle of the i.MX SSI controller
 - audio-codec : The phandle of the CS42888 audio codec
-- 
1.7.9.5

